### ISSUE TITLE
Add !default to SASS variables for local project override

### ISSUE THREAD

**Issue by owenoak** [DominicTobias/react-image-crop#151]
Hey there, great project!  I'm very excited I don't have to write this myself.  :-)

A request that you add `!default` to your SASS variables so they can be easily overridden in projects using the library.

e.g.  If the variables are declared as:

_FILE: **myproject**/node_modules/react-image-crop/lib/ReactCrop.scss_
```
$drag-handle-width: 9px !default;
$drag-handle-height: 9px !default;
...
$drag-handle-mobile-width: $drag-handle-width + 8 !default;
$drag-handle-mobile-height: $drag-handle-height + 8 !default;
...
```
Then I can override them easily in my local project's scss file before I import your SASS file:

_FILE: **myproject**/src/components/MyComponent.scss_
```
$drag-handle-width: 16px;
$drag-handle-height: 16px;
$drag-handle-mobile-width: 40px;
$drag-handle-mobile-height: 40px;
@import "../../node_modules/react-image-crop/lib/ReactCrop.scss"
...
```

Thanks again!

**Comment by DominicTobias** [comment]
Hi, thanks for the feedback I don't see a problem with adding that (added but not published yet)

**Comment by DominicTobias** [comment]
Published your changes in 3.0.9, please give it a go and let me know if there are any probs

**Comment by owenoak** [comment]
Working great!
